### PR TITLE
FashionMNIST notebook, EarlyStopping correction

### PR DESCRIPTION
**Pull request by xen0f0n**
In the example there was a single evaluator and the EarlyStopping handler was attached to it. Using a single evaluator the handler takes into account the loss for both validation and training sets and updates its best_score and counter based on both losses. Since we want the score function to rely only on the val_loss we have to attach the handler to an evaluator that loops only over the validation set.

### CODE REVIEW

**Comment by vfdev-5** [comment]
@xen0f0n thanks for the PR, yes, you are correct about that.

**Comment by vfdev-5** [comment]
@xen0f0n could you please run it to make sure that it works. Thanks

**Comment by xen0f0n** [comment]
@vfdev-5, I've ran it and it works.

**Comment by vfdev-5** [comment]
@xen0f0n thanks for the PR ! Eventually, we can now rerun it in colab too :)
